composer.json

```diff
@@ -9,8 +9,9 @@
   ],
   "licence"    : "MIT License",
   "require"    : {
-    "php"            : ">=5.6.0",
-    "symfony/framework-bundle": "^3.0.0"
+    "php"                     : ">=5.6.0",
+    "symfony/framework-bundle": "^3.0.0",
+    "doctrine/common"         : "^2.7"
```

Comment:
The master branch now contains a Composer dependency on `doctrine/annotations`, so you can remove these changes on this PR 😃.